lua/nvim-web-devicons/icons-default.lua

```diff
@@ -1316,6 +1334,18 @@ local icons_by_file_extension = {
     cterm_color = "166",
     name = "Mli",
   },
+  ["m"] = {
+    icon = "",
+    color = "#599eff",
+    cterm_color = "111",
+    name = "Tweak",
+  },
+  ["mm"] = {
+    icon = "",
+    color = "#519aba",
+    cterm_color = "74",
+    name = "Tweak",
```

Comment:
```suggestion
    name = "ObjectiveCPlusPlus",
```

Looks like you missed these two...

Reply:
Like i said i think that ill keep the icons as the original c and cpp icons but ill still set the filetype name as "tweak"
> I think I'll keep them as these languages and not "Tweaks"

> However both obj-c and obj-c++ doesn't have the icon rn so I'll have the original cpp and c icon as a replacement

Reply:
Sorry, no. They are objective C. Tweak is just a framework that uses objective C.